src/app.ts

```diff
@@ -0,0 +1,376 @@
+// Drag & Drop Interfaces
+interface Draggable{
```

Comment:
Try to create separate file for interface, helper const functions and class

Reply:
folder structure is managed